redaxo/src/core/default.config.yml

```diff
@@ -37,6 +37,8 @@ password_policy:
     digit: {min: 0}
 lang: de_de
 lang_fallback: [en_gb, de_de]
+use_https: true
+use_hsts: false
```

Comment:
So ist es super. In ein paar monaten wird es fast überall nur noch https geben und dann kann man evtl hsts per default aktivieren

Reply:
:+1: